index.html

```diff
@@ -979,33 +963,17 @@ <h4>Proof Configuration (ecdsa-jcs-2019)</h4>
 
           <ol class="algorithm">
             <li>
-Let <var>proofConfig</var> be an empty object.
-            </li>
-            <li>
-Set <var>proofConfig</var>.<var>type</var> to
-<var>options</var>.<var>type</var>.
+Let <var>proofConfig</var> be a clone of the <var>options</var> object.
             </li>
             <li>
-If <var>options</var>.<var>cryptosuite</var> is set, set
-<var>proofConfig</var>.<var>cryptosuite</var> to its value.
-            </li>
-            <li>
-If <var>options</var>.<var>type</var> is not set to `DataIntegrityProof` and
+If <var>proofConfig</var>.<var>type</var> is not set to `DataIntegrityProof` and
 <var>proofConfig</var>.<var>cryptosuite</var> is not set to `ecdsa-jcs-2019`, an
```

Comment:
As above... Does a valid proof configuration require that _both_ or _either_ be set as shown? If the former, as I'm guessing, then --
```suggestion
If <var>proofConfig</var>.<var>type</var> is not set to `DataIntegrityProof` and/or
<var>proofConfig</var>.<var>cryptosuite</var> is not set to `ecdsa-jcs-2019`, an
```